store/storetest/status_store.go

```diff
@@ -22,14 +22,11 @@ func TestStatusStore(t *testing.T, ss store.Store) {
 
 func testStatusStore(t *testing.T, ss store.Store) {
 	status := &model.Status{UserId: model.NewId(), Status: model.STATUS_ONLINE, Manual: false, LastActivityAt: 0, ActiveChannel: ""}
-
-	if err := (<-ss.Status().SaveOrUpdate(status)).Err; err != nil {
-		t.Fatal(err)
-	}
+	store.MustHaveNoErr(ss.Status().SaveOrUpdate(status))
```

Comment:
This would behave pretty similar to `require.Nil(t, ss.Status().SaveOrUpdate(status))` and `require.Nil` is simpler because you don't need another function, and is more consistent with the rest of the assertions.

Reply:
done